app-frontend/src/app/pages/admin/platform/settings/email/email.html

```diff
@@ -0,0 +1,66 @@
+<div class="content column-10">
+  <div class="dashboard-header">
+    <h1 class="h3 page-title">Email Notifications</h1>
+  </div>
+  <form name="emailSettings">
+    <div class="form-group">
+      <label for="name">User Name</label>
+      <input id="name"
```

Comment:
`autocomplete="off"` to prevent it from being filled automatically in chrome with their RF username